pandas/core/dtypes/common.py

```diff
@@ -107,9 +108,18 @@ def ensure_int64_or_float64(arr, copy=False):
     out_arr : The input array cast as int64 if
               possible without overflow.
               Otherwise the input array cast to float64.
+
+    Notes
+    -------
```

Comment:
the underline should be the same length as Notes

Reply:
done